pkg/operator/k8sutil/node.go

```diff
@@ -31,20 +32,20 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-// ValidNodeNoSched returns true if the node (1) meets Rook's placement terms,
+// validNodeNoSched returns true if the node (1) meets Rook's placement terms,
 // and (2) is ready. Unlike ValidNode, this method will ignore the
 // Node.Spec.Unschedulable flag. False otherwise.
-func ValidNodeNoSched(node v1.Node, placement cephv1.Placement) (bool, error) {
+func validNodeNoSched(node v1.Node, placement cephv1.Placement) (bool, error) {
```

Comment:
Any reason to keep the bool and not just the error? If `err != nil` the node is not valid, otherwise, we add it to the list of valid nodes.

Reply:
Yes, even better with these other changes!